src/modules/listUtils.js

```diff
@@ -0,0 +1,87 @@
+import { updateLocalStorage, getLocalStorage } from './localStorage.js';
+
+const toDoList = document.querySelector('.todo-list-ul');
+
+export const addTask = (toDoListArray, task) => {
+  toDoListArray.push({ task, completed: false, id: toDoListArray.length + 1 });
+};
+
+export const clearCompleted = (toDoListArray) => {
+  toDoListArray = toDoListArray.filter((task) => task.completed === false);
+  toDoListArray.forEach((task, index) => {
+    task.id = index + 1;
+  });
+  return toDoListArray;
+};
+
+export const renderToDoList = (toDoListArray) => {
+  toDoList.innerHTML = '';
+
+  toDoListArray = getLocalStorage();
+
+  toDoListArray.forEach((toDo) => {
+    const toDoItem = document.createElement('li');
+    toDoItem.classList.add('todo-list-li');
+
+    const toDoCheckbox = document.createElement('input');
+    toDoCheckbox.classList.add('todo-list-li-checkbox');
+    toDoCheckbox.type = 'checkbox';
+    toDoCheckbox.checked = toDo.completed;
+    toDoItem.appendChild(toDoCheckbox);
+
+    const toDoText = document.createElement('input');
+    toDoText.classList.add('todo-list-li-text');
+    toDoText.value = toDo.task;
+    toDoText.disabled = true;
+    toDoItem.appendChild(toDoText);
+
+    if (toDo.completed) {
+      toDoText.classList.add('completed');
+    }
+
+    const crossIcon = document.createElement('span');
```

Comment:
Kindly use the delete icon (such as a trash can or a dustbin) instead of a cross icon (such as an "X" mark). This is because it is generally considered to be a best practice in user interface design. This is because the delete icon more clearly communicates the action that will take place when the user interacts with it.